src/app/app.component.ts

```diff
@@ -14,7 +17,21 @@ export class AppComponent implements OnInit {
     private router: Router,
     private translateService: TranslateService,
     private storeUrlService: StoreUrlService
-  ) {}
+  ) {
+    if (environment.applicationInsightsConnectionString) {
+      const angularPlugin = new AngularPlugin();
+      const appInsights = new ApplicationInsights({
+        config: {
+          connectionString: environment.applicationInsightsConnectionString,
+          extensions: [angularPlugin],
+          extensionConfig: {
+              [angularPlugin.identifier]: { router: this.router }
+          }
+        }
+      });
```

Comment:
The docs at:
https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript-angular-plugin
mention the `instrumentationKey` property. Where did you get the current options from?

Reply:
`instrumentKey` is deprecated in favor of `connectionString`, as stated here:
https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript

Reply:
OK, it looks like they forgot to update the Angular docs then. Thanks!